website/src/components/Layout.tsx

```diff
@@ -52,7 +55,12 @@ export default class Layout extends React.PureComponent<LayoutProps> {
                     <meta property="og:image" content={metaProps.image} />
                     <meta property="og:description" content={metaProps.description} />
 
-                    <meta name="description" content={metaProps.description} />
+                    {metaProps.seoDescription ? (
+                        <meta name="description" content={metaProps.seoDescription} />
+                    ) : (
+                        <meta name="description" content={metaProps.description} />
+                    )}
```

Comment:
Can be simplified :)
```suggestion
                    <meta name="description" content={metaProps.seoDescription || metaProps.description} />
```